classify.py

```diff
@@ -14,6 +14,8 @@
                     help='List of comma separated file extensions to filter by when operating on a directory. Default: tif,tiff,jp2,jpg,jpf,jpx,png'),
 parser.add_argument('-l', '--file-list', action="store_true",
                     help='If provided, then the src_path will be treated as a text file containing a list of newline separated paths to normalize.'),
+parser.add_argument('-r', '--restart', action="store_true",
```

Comment:
It might worth noting that the report csv is _not_ reset when this flag is enabled, as the truncation of new results to old ones is probably not the desired effect for someone testing (different models?) repeatedly on the same data.